source/_components/weather.environment_canada.markdown

```diff
@@ -0,0 +1,45 @@
+---
+layout: page
+title: "Environment Canada Weather"
+description: "Weather from Environment Canada."
+date: 2017-07-07 18:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: environment_canada.jpg
+ha_category: Weather
+ha_release: 0.74
+ha_iot_class: "Cloud Polling"
+---
+
+The `environment_canada` weather platform uses [Environment Canada](https://weather.gc.ca/mainmenu/weather_menu_e.html) as a source for current meteorological data.
+
+- The platform automatically determines which weather station to use, according to this hierarchy:
+  - Station code from sensor configuration
+  - Closest station to latitude / longitude from sensor configuration
+  - Closest station to latitude / longitude from Home Assistant core configuration
+
+- If no name is given, the weather entity will be named `weather.<station_name>`.
+- The sensor checks for new data every 10 minutes, and the source data is typically updated hourly within 10 minutes after the hour.
+
+To add Environment Canada weather to your installation, add the desired lines from the following example to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+weather:
+  - platform: environment_canada
+    latitude: 50
+    longitude: -100
+    station: ON/s0000430
+    name: Ottawa
+```
+
+Configuration variables:
```

Comment:
Please use the configuration tags, for more information please see:
https://home-assistant.io/developers/documentation/create_page/#configuration